MapboxCoreNavigation/sequence.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+extension Sequence where Element: Hashable {
+    #if !swift(>=4.1)
```

Comment:
Strangely, `#if swift(<4.1)` does not work.

Reply:
Yeah, it's silly, but 🤷‍♂️

Reply:
Should this file name have a capital S ?